.vuepress/config.js

```diff
@@ -81,10 +81,18 @@ module.exports = {
           children: [
             '/learn/advanced/asynchronous_updates',
             '/learn/advanced/snapshots_vs_dumps',
+            '/learn/advanced/bucket_sort',
+            '/learn/advanced/concat',
+            '/learn/advanced/datatypes',
+            '/learn/advanced/indexation',
+            '/learn/advanced/prefix',
+            '/learn/advanced/storage',
+            '/learn/advanced/tokenization',
+            '/learn/advanced/typotolerance',
```

Comment:
What do you think of keeping these within a subsection in the sidebar, so they're not grouped together with all other advanced guides? My reasoning is that I would consider these less essential for most users and more advanced than the other ones in this category.

```suggestion
            {
              title: 'Under the Hood',
              collapsable: false,
              children: [
                '/learn/advanced/bucket_sort',
                '/learn/advanced/concat',
                '/learn/advanced/datatypes',
                '/learn/advanced/indexation',
                '/learn/advanced/prefix',
                '/learn/advanced/storage',
                '/learn/advanced/tokenization',
                '/learn/advanced/typotolerance'
              ]
            }
```

Reply:
I hesitated putting everything under Advanced, but in the end I wasn't sure about what we gain from separating them?

Trying to think from the user's perspective, what benefit would they get from having `Advanced` and `Under the hood` categories/subsections? Is the distinction you indicate (essential advanced vs non-essential advanced articles) important to the user and, if so, is `Under the hood` the best way to communicate that?

Perhaps I'm just not a fan of the `Under the hood` heading because I'm not 100% sure what I will find under it?

Reply:
I think it is a useful distinction for users. It's also a useful reminder for us, since most of the articles in Under the Hood have not been updated in a long time and could really use some love (I'm looking at you `datatypes.md`), and some are just shorter and lower quality than others on the site (e.g. `bucket_sort.md`, which is basically an entire page for an extremely simplified definition of bucket sort).

Obviously the ultimate goal is to make all of these articles as high quality as anything else on the site, but I think that's out of the scope of the current restructure—we will already have our hands full with Feature References—so for now we should focus on what we can do purely w/ content organization. For me, that means we should try to keep them "together" and separate from most articles. (And please let's just get rid of that bucket sort page. It's not adding anything. Links to it could just as easily go to wikipedia.)

Totally open to renaming the section. Ideas? I've got:

- MeiliSearch internals
- How stuff works
- Nuts & bolts

Reply:
I agree with keeping all of this under a separate subsection. It's advanced but users don't need to know all of this to use MeiliSearch (maybe minus the [space separators](https://docs.meilisearch.com/reference/under_the_hood/datatypes.html#string)).  
Inner workings sounds great!

Reply:
`Inner workings` subsection it is, then!